src/Form/atoms/RadioButton.tsx

```diff
@@ -0,0 +1,124 @@
+import React, {useCallback, InputHTMLAttributes } from 'react';
+import styled, { css } from 'styled-components';
+
+const InnerRadio = styled.div`
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  user-select: none;
+`;
+
+const OuterRadio = styled.label<{ isChecked: boolean, disabled: boolean}>`
```

Comment:
Hi! I have removed the label from this component and made an update in the Label component to have rightAlign prop.

The left margin is fix in Label is fix  to 12px but the margin bellow was actually created with a Wrapper.
Please let me know if this needs to be re-updated.

<img width="214" alt="Screen Shot 2021-09-21 at 19 09 32" src="https://user-images.githubusercontent.com/10409078/134152860-e8152345-449b-4e14-8cfe-e7111f277c9b.png">

Reply:
I have not updated the checkbox since I see that is using the label's "onChange" to control the checkbox value and I'm not using this component in this PR.

Also I see that there is a follow up issue for it https://github.com/future-standard/scorer-ui-kit/issues/123

<img width="635" alt="Screen Shot 2021-09-21 at 19 24 37" src="https://user-images.githubusercontent.com/10409078/134154909-bec0da6d-3081-43f9-997a-4a467b9b26ce.png">

<img width="1294" alt="Screen Shot 2021-09-21 at 19 24 47" src="https://user-images.githubusercontent.com/10409078/134154940-e957db56-f63c-456d-afdd-94ad01e7fa09.png">